target/linux/ramips/mt7620/base-files/etc/board.d/01_leds

```diff
@@ -154,6 +154,7 @@ netgear,wn3000rp-v3)
 	set_wifi_led "$boardname:green:router"
 	;;
 netgear,ex3700|\
+netgear,ex6100-v1|\
```

Comment:
dts trigger for wireless can be used instead

Reply:
Actually it's not WiFi LED, it indicates the connection to router (WAN) and clients (LAN). I just see EX6130 is here so I add this device too.

Reply:
```
	ucidef_set_led_netdev "wlan5g" "ROUTER (green)" "$boardname:green:router" "wlan0"
	ucidef_set_led_netdev "wlan2g" "DEVICE (green)" "$boardname:green:device" "wlan1"
```

These are WiFi LEDs.

> I just see EX6130 is here so I add this device too.

So you are testifying that you just copy/paste from EX6130 no matter whether it is required or makes any sense, because you just don't check what you copy at all?

Reply:
These are also not WiFi LEDs on EX3700 and EX6130. Both are link status LEDs.
![image](https://user-images.githubusercontent.com/12028138/75083270-e7089500-5552-11ea-8ff8-8461d1927ae1.png)
![image](https://user-images.githubusercontent.com/12028138/75083285-f38ced80-5552-11ea-8552-987da0f2537c.png)
I just think it's accepted on two devices, so I add this device too.

Reply:
https://openwrt.org/docs/guide-user/hardware/led/generic.rssileds
You can make use of rssileds to show signal strength